can_msgs/CMakeLists.txt

```diff
@@ -1,54 +0,0 @@
-cmake_minimum_required(VERSION 3.5)
```

Comment:
Earlier this week I ran into some issues with removing this package. Can we leave it in and just not create a mapping for it? I don't think we'll ever need to bridge a raw can topic.

Reply:
it was causing issues when i was building i can investigate more on this.